docs/runtimes/mlrun_code_annotations.ipynb

```diff
@@ -0,0 +1,1253 @@
+{
```

Comment:
<p>MLRun annotations are used to identify the code that needs to be converted into an MLRun function. They provide non-intrusive hints which indicate which parts of your notebook should be considered as the code of the function. Annotations start a code block using <code># mlrun: start-code</code> and end a code block s , with <code># mlrun: end-code</code>. Use the <code>#mlrun: ignore</code> to exclude items from the code qualified annotations. Make sure that the annotations include anything required for the function to run.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/1456/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>